static/app/views/performance/transactionSummary/transactionEvents/utils.tsx

```diff
@@ -0,0 +1,33 @@
+import {Query} from 'history';
+
+export function generateEventsRoute({orgSlug}: {orgSlug: String}): string {
+  return `/organizations/${orgSlug}/performance/summary/events/`;
+}
+export function eventsRouteWithQuery({
+  orgSlug,
+  transaction,
+  projectID,
+  query,
+}: {
+  orgSlug: string;
+  transaction: string;
+  query: Query;
+  projectID?: string | string[];
+}) {
+  const pathname = generateEventsRoute({
+    orgSlug,
+  });
```

Comment:
nit: Since it seems like `generateEventsRoute` isn't being used anywhere else
```suggestion
  const pathname = `/organizations/${orgSlug}/performance/summary/events/`;
```

Reply:
👍